sw/device/silicon_creator/mask_rom/mask_rom_epmp.h

```diff
@@ -7,6 +7,7 @@
 
 #include <stdint.h>
 
+#include "sw/device/silicon_creator/lib/drivers/lifecycle.h"
```

Comment:
Can you update the Bazel rules to make them aware of this new dependency? I think it's excluded from the sandbox and causing problems building the mask_rom_epmp_test otherwise.

```bazel test //sw/device/silicon_creator/mask_rom:verilator_mask_rom_epmp_test```

Reply:
Updated